### DISCUSSION TITLE
How to enable/use multiple exposures after version 4.0 ?

### DISCUSSION THREAD

**Discussion by Nevsksar** [sn4k3/UVtools#806] [Q&A] [answered]
It has been a while since i used another resin and had to do an exposure test. So I went as usual to create my exposure test but could not find the multiple exposure table/options. (My printer as support for it (elegoo saturn) and I always used it.)

So I went through the changelog of the latest versions and found this regarding verion 4.0 : 
"Exposure time finder:
(Fix) "Multiple layer height" panel visibility was wrongly check for ExposureTime instead of PositionZ
(Fix) "Multiple exposures" panel was always visible even if not able
(Improvement) Disable "Multiple exposure" and "Multiple heights" for the AnyCubic format"

So I rolled back to version 3.15.1 to see if this was the case, and there it was to regular multiple exposure settings again.

Now I raise the question, how do I use the multiple exposures setting on the newer versions ? they do not show up as an option in the exposure time finder, is there anything new i need to set up for it to appear ?

**Comment by sn4k3** [comment, marked as answer]
IF they don't show up mean not available on your config. Check your file, must be CTBv3 or greater.

**Reply by Nevsksar** [reply]
Thats it ! however it used to run fine with ctb v2 (Standard lychee setting for a Saturn) so don't know if this behaviour is fully intented.

**Reply by sn4k3** [reply]
While CTBv2 have parameters it rarely used by most printers, hence it is disabled as having not.
